packages/radio-player/.vscode/settings.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
curious on this addition?  i guess the yarn start wanted it?

Reply:
oh shoot! I added these in by mistake so sorry & thanks for the catch